ibis/expr/operations.py

```diff
@@ -2305,6 +2305,13 @@ class Strftime(ValueOp):
     output_type = rlz.shape_like('arg', dt.string)
 
 
+class StringToTimestamp(ValueOp):
+    arg = Arg(rlz.string)
+    format_str = Arg(rlz.string)
+    timezone = Arg(rlz.string, default=None)
+    output_type = rlz.shape_like('arg', dt.timestamp)
```

Comment:
I think this should be `dt.Timestamp(timezone='UTC')` since BigQuery returns timestamps in UTC. Other backends that implement this can follow suit.